static/css/inline-selection.scss

```diff
@@ -491,6 +505,7 @@ $very-light-grey: #ededed;
   text-align: center;
   padding-top: 20px;
   width: 400px;
+  user-select: none;
```

Comment:
This is super important. I think it fell out of our CSS at some point. we really need to land this line.